gke-deploy/go.mod

```diff
@@ -1,58 +1,17 @@
-module github.com/GoogleCloudPlatform/cloud-builders/gke-deploy
+module github.com/GoogleCloudPlatform/cloud-builders
```

Comment:
This is probably also a bad idea, since it will likely conflict with gcs-fetcher's go.mod in confusing ways.

Reply:
Sorry, not sure why this is happening when I run my `update-deps.sh` script

Reply:
^ Fixed by adding `gke-deploy` to the end of that url. I'm testing gcloud builds submit right now before pushing.

Reply:
Fixed